### DISCUSSION TITLE
Support for C#/DotNet (Core)

### DISCUSSION THREAD

**Discussion by ThaDaVos** [spatie/ray#475] [Ideas]
Loving the idea of Ray but my stack mostly exists of:
1. VueJs (supported)
2. C#/Asp.net (core) (unsupported)

It would be amazing if the C# based projects or dotnet (core) based projects could also profit by the amazing Ray

**Comment by AdrianMrn** [comment]
We don't have any experience with C# or .NET, but feel free to give it a go yourself! This shouldn't be too difficult or take up too much time: https://spatie.be/docs/ray/v1/developing-ray-libraries/developing-a-ray-library

**Reply by mathmul** [reply]
@ThaDaVos did you make it?

**Reply by ThaDaVos** [reply]
@mathmul - no, used Ray for a year and eventually stopped using it cause I didn't use it that much
